src/main/java/dk/cngroup/lentils/service/TeamService.java

```diff
@@ -31,14 +31,26 @@ public TeamService(final TeamRepository teamRepository,
         this.roleService = roleService;
     }
 
+    public Team update(final Team team) {
+        team.setPin(getUniquePin());
```

Comment:
Tohle teda pri úpravě týmu vygeneruje nový PIN. To je správně, jo?

Reply:
nevim jestli je to spravne, ale takto se to dělalo na začátku jeste nez jsem k lusteninam prisel ja :)
Při uprave se generoval vzdy novy pin -> tak jsem to tak nechal

Reply:
ale klidne se pri uprave muze pin nechat a zmenit jen data teamu - jmeno a počet - zalezi na nas :)

Reply:
upraveno